frontend/src/components/ClusterModal.js

```diff
@@ -92,7 +92,8 @@ const ClusterModal = ({
   };
 
   // Reload annotations whenever there's a new result set
-  useEffect(async () => {
```

Comment:
I put async function inside a sync one in useEffect.
    
    This is what useEffect expects and it's much better practice:
    https://ultimatecourses.com/blog/using-async-await-inside-react-use-effect-hook